fact_extractor/plugins/unpacking/srec/code/srec.py

```diff
@@ -32,6 +36,11 @@ def unpack_function(file_path, tmp_dir):
     return {'output': 'Successfully decoded srec file'}
 
 
+def _get_unpacked_filename(file_path):
+    file_name = f'{Path(file_path).name}.bin'
```

Comment:
Can you not do this with the `.with_suffix()` and the `.stem` attributes? Sth. like 
```python
return Path(file_path).stem.with_suffix('.bin')
```

Reply:
changed